src/components/LoginPage/LoginForm.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="login-form">
+    <h4 class="title">Masuk ke Akunmu</h4>
+    <v-text-field v-model="email" label="Email" class="input-text" />
+    <v-text-field
+      v-model="password"
+      :append-icon="showPassword ? 'fa-eye' : 'fa-eye-slash'"
+      :rules="[rules.required, rules.min]"
+      :type="showPassword ? 'text' : 'password'"
+      name="input-10-1"
+      label="Password"
+      hint="At least 8 characters"
+      counter
+      @click:append="showPassword = !showPassword"
+      class="input-text"
+    />
+    <v-btn color="teal darken-1" min-width="350" dark>Masuk</v-btn>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'LoginForm',
+  data() {
+    return {
+      email: '',
+      password: '',
+      showPassword: false,
+      rules: {
+        required: value => !!value || 'Required.',
+        min: v => v.length >= 6 || 'Min 6 characters',
+        emailMatch: () => "The email and password you entered don't match",
```

Comment:
If `emailMatch` is not used by any methods or in HTML's component, then I think you should remove it.

Reply:
right now, we don't need this because there is no validate function in LoginForm.vue . Maybe next time, we'll need this. But, for now, I will remove emailMatch